src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/GeneratedDocumentSynchronizerTest.cs

```diff
@@ -49,9 +45,6 @@ public void DocumentProcessed_UnknownVersion_Noops()
     [Fact]
     public void DocumentProcessed_KnownVersion_Publishes()
     {
-        // Arrange
-        _cache.TrackDocumentVersion(_document, version: 1337);
```

Comment:
Is this test still meaningful without this set up?

Reply:
It got renamed, and a couple more added so my answer is: yes/no.